ibm/service/power/data_source_ibm_pi_volume.go

```diff
@@ -152,6 +164,11 @@ func DataSourceIBMPIVolume() *schema.Resource {
 				Set:         schema.HashString,
 				Type:        schema.TypeSet,
 			},
+			Attr_VolumeName: {
```

Comment:
```suggestion
			Attr_Name: {
```
I think this should just be name